.gitignore

```diff
@@ -2,6 +2,7 @@
 build
 node_modules
 config/local.json
+packages/devtools-config/configs/local.json
```

Comment:
I've removed that line from the gitignore file. Let me know if there's anything else I need to do.